src/components/Input/Input.tsx

```diff
@@ -41,7 +41,7 @@ const Input = ({
       {icon !== undefined && (
         <FontAwesomeIcon
           icon={icon}
-          className={`absolute left-2 top-3.5 text-gray-400 ${size}`}
+          className={`absolute left-2 top-3.5 text-gray-400 h-5 w-5 ${size}`}
```

Comment:
I see we also have the `size` prop here, used as class name. Does it still work as expected?

Looking at the props I see that size is:

```
size: "small" | "medium" | "large";
```

But now you added fixed width and height here. Check the storybook for that component to see that it behaves accordingly